test/hybig/HyBIG_Regression.ipynb

```diff
@@ -278,6 +290,87 @@
     "    print('Skipping test: HyBIG regression tests not configured for this environment.')"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "id": "ea255f34",
+   "metadata": {},
+   "source": [
+    "## Failure cases raised by bad input parameters."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e59199d8",
+   "metadata": {},
+   "source": [
+    "### Test: Scale Extent has parameters in wrong order.\n",
+    "xmin > xmax"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3d8a3116",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "if environment_information is not None:\n",
+    "    \n",
+    "    reversed_xscale_extent = [23, 0, 22, 1]\n",
+    "    aster_request = Request(collection=environment_information['aster_collection'],\n",
+    "                            granule_id=environment_information['aster_granules'][0],\n",
+    "                            scale_extent=reversed_xscale_extent,\n",
+    "                            crs='EPSG:4326',\n",
+    "                            format='image/jpeg')\n",
+    "\n",
+    "    aster_job_id = harmony_client.submit(aster_request)\n",
+    "    try: \n",
+    "        harmony_client.wait_for_processing(aster_job_id, show_progress=True)\n",
+    "        print_error('Exception not rasied for bad Scale Extents')\n",
+    "        assert False, 'Fail Scale Extent test'\n",
+    "    except Exception as exception:\n",
+    "        assert 'Harmony ScaleExtents must be in order [xmin,ymin,xmax,ymax]' in str(exception), 'Exception not raised correctly'\n",
+    "        print_success('Exception Raised Correctly for Scale Extents')\n",
+    "else:\n",
+    "    print('Skipping test: HyBIG regression tests not configured for this environment.')        \n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "e8deff1f",
+   "metadata": {},
+   "source": [
+    "### Test: Scale Sizes are Positive\n"
```

Comment:
I got a failure here (I tried it a couple of times):

```
HTTPError: 400 Client Error: Bad Request for url: http://localhost:3000/C1256584478-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?forceAsync=true&outputcrs=EPSG%3A4326&scaleSize=0.002%2C-0.002&granuleId=G1256584570-EEDTEST&format=image%2Fjpeg
```

Reply:
Interesting, what happens if you just visit that url in the failure? 

``` shell
http://localhost:3000/C1256584478-EEDTEST/ogc-api-coverages/1.0.0/collections/all/coverage/rangeset?forceAsync=true&outputcrs=EPSG%3A4326&scaleSize=0.002%2C-0.002&granuleId=G1256584570-EEDTEST&format=image%2Fjpeg
```

Because this particular test is actually failing in harmony, not HyBIG, but I left the test in anyway.  This is related to the question, `should Harmony have a validation check for scaleExtent?` See comments on the ticket.


I have the latest and greatest harmony-in-a-box and this is what I see when I visit the URL:


<img width="933" alt="Screenshot 2023-10-19 at 8 53 19 AM" src="https://github.com/nasa/harmony-regression-tests/assets/479480/e58dc20b-4d6d-47eb-8327-3fa5ea07cdb4">

Reply:
I did see the comments in the ticket, yup. If this is testing Harmony-core functionality, it probably lives in one of the Harmony-core regression test notebooks.

I also have the latest Harmony-in-a-box. The URL fails when placed directly in the browser, with the error that you have in this comment. But I continue to get this other error when using `harmony-py`, which likely means that `harmony-py` is doing something additional here in validating the URL before it gets sent to Harmony, maybe. Do you see different behaviour when running the request via `harmony-py`?

Reply:
Using harmony-py, like the test?  It passes for me. Raises the expected error.

Reply:
As discussed in Slack - differences here were due to my local environment having an older version of `harmony-py`. I get this test to pass now.